packages/builder/src/builtin-tasks/node-info.ts

```diff
@@ -0,0 +1,21 @@
+import { task } from "../internal/core/config/config-env";
+import { AlgobRuntimeEnv, TaskArguments } from "../types";
+import { TASK_NODE_INFO } from "./task-names";
+
+const algosdk = require("algosdk");
+
+export default function (): void {
+  task(TASK_NODE_INFO, "Prints node info and status")
+    .setAction(nodeInfo);
+}
+
+async function nodeInfo(_taskArgs: TaskArguments, env: AlgobRuntimeEnv) {
+  console.log("IN THE TASK", env.network);
```

Comment:
Is this a debug comment?

Reply:
yes, the task is not fully done, but the functionality is already there.